lib/ReadsUtils/ReadsUtilsImpl.py

```diff
@@ -823,15 +839,75 @@ def upload_reads(self, ctx, params):
         self.log('Starting upload reads, parsing args')
```

Comment:
I don't think we should delete the 'ctx' variable (might have side effects?).  I'm not sure that we can actually get rid of this pep8 violation because 'ctx' is in the generated code, so we might just have to leave this as is.  Adding a comment to quiet the warning also isn't really possible because the warning probably appears in the generated code.